tickets/src/components/content/purchase/TicketCode.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import QRCode from 'qrcode.react'
+
+export const TicketCode = ({ signedAddress, publicKey }) => {
+  if (!signedAddress || !publicKey) {
+    return null // If we don't have a signed address or public key, we can't return a QR code
+  } else {
+    const validateUri =
+      'https://tickets.unlock-protocol.com/checkin/' +
```

Comment:
Let's not hard code URL, it makes doing things locally really hard? Can we use a config?